config/routes.rb

```diff
@@ -11,6 +11,8 @@
     mount Sidekiq::Web => '/sidekiq'
   end
 
+  resources :exhibit_finder, only: :show
```

Comment:
I could feature flag this route if we wanted to keep it out of prod, but it seems relatively harmless.